### DISCUSSION TITLE
Error while trying to create special repository

### DISCUSSION THREAD

**Discussion by akankshadash** [community/community#78901] [Repositories] [answered]
### Select Topic Area

Question

### Body

Unable to create special repository as it says:
![image](https://github.com/community/community/assets/61039826/082d3e8a-b95c-4af8-b9b7-5d49824e92bd)

please help me out to solving this error

**Comment by Kiran1689** [comment]
Hi @akankshadash 

You already created a repo with that name `akankshadash`, Please check your repositories you will find it. You can't create a another repo with the same repo name on Github.

**Reply by Kiran1689** [reply, marked as answer]
As I said you already created your special repo with your username.

Here is your special repo: https://github.com/akankshadash/akankshadash

In this repo create README.md file and write content.. which should be shown on your GitHub profile.

**Reply by akankshadash** [reply]
Thank you .. checked my repo i had already with the same name
